routes/challenges.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express')
+const router = express.Router()
+const authenticate = require('../middlewares/authenticate')
+const challengesController = require('../controllers/challengeController')
+
+/**
+ * @swagger
+ * /challenges/:
```

Comment:
Any specific reason to add a ` / ` after ` /challenges ` ?  Got a bit confused here, rest great work!! 💯

Reply:
I didn't add it, I just forgot to remove it 😅 
Thanks for informing though

Reply:
okay! 👍